Tests/autoload.php.dist

```diff
@@ -0,0 +1,31 @@
+<?php
+
+$vendorDir = __DIR__.'/../vendor';
+require_once $vendorDir.'/symfony/src/Symfony/Component/ClassLoader/UniversalClassLoader.php';
+
+use Symfony\Component\ClassLoader\UniversalClassLoader;
+
+$loader = new UniversalClassLoader();
+$loader->registerNamespaces(array(
+    'Symfony'                => array($vendorDir.'/symfony/src', $vendorDir),
+    'Doctrine\\Common'       => $vendorDir.'/doctrine-common/lib',
+    'Doctrine\\DBAL'         => $vendorDir.'/doctrine-dbal/lib',
+    'Doctrine'               => $vendorDir.'/doctrine/lib',
+    'DoctrineExtensions'     => $vendorDir.'/beberlei-doctrine-extensions/lib',
+));
+$loader->registerPrefixes(array(
+    'Twig_' => $vendorDir.'/twig/lib',
+));
+$loader->register();
+
+spl_autoload_register(function($class) {
+    if (0 === strpos($class, 'Kitpages\\DataGridBundle\\')) {
```

Comment:
I see this bundle is not even required in `composer.json`

Reply:
Bad copy paste :)